packages/zk-certificates/tasks/revokeZkCertificate.ts

```diff
@@ -59,28 +59,28 @@ async function main(args: any, hre: HardhatRuntimeEnvironment) {
 
   console.log(
     chalk.green(
-      `Revoked the zkKYC certificate ${args.leafHash} on-chain at index ${
+      `Revoked the zkCertificate ${args.leafHash} on-chain at index ${
         args.index as number
       }`,
     ),
   );
 }
 
 task(
-  'revokeZkKYC',
-  'Task to revoke a zkKYC certificate with leaf hash and merkle tree',
+  'revokeZkCertificate',
```

Comment:
To remember: we also have to adjust the documentation in https://github.com/Galactica-corp/Documentation/blob/master/kyc-guardian-guide/create-and-issue-zkkyc.md